### DISCUSSION TITLE
Serilog switch from File to Db - Help needed!

### DISCUSSION THREAD

**Discussion by helptiger** [mysql-net/MySqlConnector#1221] [Q&A] [answered]
Hi,
in a Console project I use Serilog quite simply to write to a log file:

<code>Logger log = new LoggerConfiguration().WriteTo.File("log.txt", rollingInterval: RollingInterval.Day).CreateLogger();</code>

In the project I also use MySqlConnector and would like to use logging as well.
Unfortunately the documentation is not enough for me to get that done somehow.
**Has anyone already implemented this and can show me a simple solution with code to make it work?**

I would be very grateful!

Thanks

**Comment by bgrainger** [comment, marked as answer]
Are you wanting to connect MySqlConnector logging to Serilog, or write all Serilog output to a MySQL database, or both?

To send MySqlConnector logging to Serilog, see https://mysqlconnector.net/overview/logging/#serilog.

To send Serilog logging to a MySQL database, see this project: https://github.com/TeleSoftas/serilog-sinks-mariadb. (Note, I've never used it, but it depends on MySqlConnector, so it seems like the most appropriate choice.)

**Reply by helptiger** [reply]
Bradley, thanks for your comment. The 2nd link (https://github.com/TeleSoftas/serilog-sinks-mariadb) brought me to the solution. Thanks for your support and your connector, I have been using it happily for several years.
